less/index.less

```diff
@@ -54,6 +53,97 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+}
+
+h1 {
+	font-size:4rem;
+	// margin-bottom:@butt-pillow;
+}
+h2 {
+	font-size:2rem;
+	color:gray;
+	padding:20px;
+}
+
+ul {
+	list-style-type:circle;
+	li {
+		padding:1rem;
+		line-height:1.25;
+		&:hover {color:red;}
+	}
+}
+
+b {
+	font-weight:bold;
+	font-size: 1.25rem;
+}
+
+// VARIABLES
+@butt-pillow:20px;
+
+// MIXINS
+.column-center() {
+	display:flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+// STYLES BEGIN
+.container {
+	border-left:1px solid black;
+	border-right:1px solid black;
+	margin:0 auto;
+	max-width:777px;
+	padding:3%;
+	.column-center();
+}
```

Comment:
take another look at the MVP requirements, you want to have all of your page's element-specific styles within your `.container` block -- this should just be a simple matter of rearranging where your css blocks are to stuff them inside this bracket